src/components/sbb-loading-indicator/sbb-loading-indicator.scss

```diff
@@ -0,0 +1,235 @@
+@use '../../global/styles' as sbb;
```

Comment:
General feedback: can we use, where possible, nested rules and vars to improve readability/repetitions?

Reply:
I find it more readable too, but I remember from a previous review with @jeripeierSBB that we should avoid to nest them

Reply:
I think here we can do a little bit more of nesting. At least as long all spans don't have any specific name, it would be more readable if the definitions are nested I think. But yes generally we should avoid nesting as much as possible :)